src/lib/sub-app/sub-app.factory.ts

```diff
@@ -279,9 +279,8 @@ function moveDirectoryTo(
   tree.getDir(srcDir).visit((filePath: Path, file: Readonly<FileEntry>) => {
     const newFilePath = join(destination as Path, filePath);
     tree.create(newFilePath, file.content);
-    tree.delete(filePath);
+    tree.delete(srcDir);
```

Comment:
Even though it may fix the reported issue, are we sure that calling `tree.delete(srcDir);` X times makes sense?

Reply:
indeed

although we're calling `tree.delete` several times, looks like angular schematics is treating such usage:

![image](https://github.com/nestjs/schematics/assets/13461315/f496f7df-e937-4e91-8f17-6422144460ae)

I'm still trying another way to achieve that. `tree.exists` didn't worked, just the usual try-catch on `tree.delete`